src/app/core/top-panel/top-panel.component.ts

```diff
@@ -83,7 +84,7 @@ export class TopPanelComponent implements OnInit {
       if (organization && project) {
         this.store.dispatch(new OrganizationsAction.Select({organizationId: organization.id}));
         this.store.dispatch(new ProjectsAction.Select({projectId: selectProject ? project.id : null}));
-        this.router.navigate(['/workspace']);
+        this.store.dispatch(new RouterAction.Go({path: ['workspace']}))
```

Comment:
You don't have to use this action in components. It is there for navigating inside the store where you change one type of action to another. We don't need to use actions for everything.